runtime/foucoco/Cargo.toml

```diff
@@ -192,7 +192,6 @@ std = [
     "pallet-xcm/std",
     "parachain-info/std",
     "parachain-staking/std",
-    "orml-currencies-allowance-extension/std",
```

Comment:
Why are we removing this?

Reply:
This was added twice.
It's a duplicate of [this line](https://github.com/pendulum-chain/pendulum/blob/58d806850b912e22905c444d99db7a9b0757df98/runtime/foucoco/Cargo.toml#L240).